website/docs/components/DataVisualization/stat.mdx

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
📖  is this a new part of the process when creating a new package @alexanderchan ? do i have do to something or will it be auto deployed?

Reply:
Actually I'm not sure either, I'm guessing @wesleycole probably hooked up netlify to deploy changes to master to the docs but we can check with him when he's back.